src/main/groovy/nextflow/executor/AwsBatchExecutor.groovy

```diff
@@ -359,7 +364,7 @@ class AwsBatchFileCopyStrategy extends SimpleFileCopyStrategy {
         if( normalized ) {
             result << ""
             normalized.each {
-                result << "nxf_s3_upload $it s3:/${targetDir.resolve(it)} || true" // <-- add true to avoid it stops on errors
+                result << "nxf_s3_upload $it s3:/${targetDir} || true" // <-- add true to avoid it stops on errors
             }
```

Comment:
This should be: 

```
nxf_s3_upload '$it' s3:/${targetDir}/ || true
```

Not sure, but I think the ending slash in the target path is required

Reply:
Yes sounds feasible, I'll do some tests and then add to this pull request.